gems/pending/spec/vmware/vmware_vcloud_event_monitor_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'vmware/vmware_vcloud_event_monitor'
+
+describe VmwareVcloudEventMonitor do
+  before do
+    @original_log = $log
+    $log = double.as_null_object
```

Comment:
We're suppressing log in spec test (`before do` block sets it to nil, `after do` restores it) to keep original log clean.
The idea is copy-pasted from [here](https://github.com/ManageIQ/manageiq/blob/master/gems/pending/spec/openstack/events/openstack_rabbit_event_monitor_spec.rb#L6).